docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -534,17 +534,43 @@ Migrating Ansible content to a different collection
 
 First, look at `Ansible Collection Checklist <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
 
-To migrate content from one collection to another, you need to create three PRs as follows:
+To migrate content from one collection to another, if the collections are parts of `Ansible distribution <https://github.com/ansible-community/ansible-build-data/blob/main/2.10/ansible.in>`_:
 
-#. Create a PR against the old collection to remove the content.
-#. Create a PR against the new collection to add the files removed in step 1.
-#. Update the ``ansible/ansible:devel`` branch entries for all files moved.
+#. Copy content from the source (old) collection to the target collection.
+#. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. If the target collection has been released with the new content as a new minor version for the same major version that is currently included in Ansible, this ensures that users can resolve the deprecation warning without having to install something manually that potentially breaks backwards compatibility.
```

Comment:
The deprecation should only be released once a compatible new version of the target collection has been released which contains the content.

Reply:
changed. PTAL